src/pages/Options/Home/utils.ts

```diff
@@ -5,22 +5,20 @@ import { OptionsMarkets } from '../types';
 
 export const sortOptionsMarkets = (markets: OptionsMarkets, synthsMap: SynthDefinitionMap) =>
 	orderBy(
-		markets
-			.filter((optionsMarket) => optionsMarket.isOpen)
-			.map((optionsMarket) => {
-				const { phase, timeRemaining } = getPhaseAndEndDate(
-					optionsMarket.biddingEndDate,
-					optionsMarket.maturityDate,
-					optionsMarket.expiryDate
-				);
+		markets.map((optionsMarket) => {
+			const { phase, timeRemaining } = getPhaseAndEndDate(
+				optionsMarket.biddingEndDate,
+				optionsMarket.maturityDate,
+				optionsMarket.expiryDate
+			);
 
-				return {
-					...optionsMarket,
-					phase,
-					asset: synthsMap[optionsMarket.currencyKey]?.asset || optionsMarket.currencyKey,
-					timeRemaining,
-					phaseNum: PHASE[phase],
-				};
-			}),
+			return {
+				...optionsMarket,
+				phase,
+				asset: synthsMap[optionsMarket.currencyKey]?.asset || optionsMarket.currencyKey,
```

Comment:
when you use the ? key you should use ?? instead of ||
so with destructuring 
synthsMap[currencyKey]?.asset ?? currencyKey,

Reply:
Hmm, that's not really true, I mean the optional chaining has nothing to do with `??`.

`??` is checking for null/undefined and `||` is checking for all the falsy values (0, false, empty string, etc). It could definitely work here... but I don't think its a mandatory change since in this case we know it's either a string or undefined... so `||` would work the same as `??`